src/bot.ts

```diff
@@ -39,7 +39,7 @@ export class DiscordBot {
     this.bridge = bridge;
   }
 
-  get ClientFactory(): DiscordClientFactory {
+  public getClientFactory(): DiscordClientFactory {
```

Comment:
What's the meaning of this change? I prefer to use the native get/set methods over pure functions?